language_detector/main.py

```diff
@@ -1,8 +1,21 @@
 # -*- coding: utf-8 -*-
 
-"""This is the entry point of the program."""
-
 def detect_language(text, languages):
-    """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    
+    globalcount = 0
+    lang = ""
+    
+    for dictionary in languages:
+        
+        count = 0
+        for word in text.split():
```

Comment:
you know, actually i made that same mistake when i was initially crafting my solution to add my groups work, so i figured it out then and had to change it from 
```python
text.split().lower()
``` 
to 

```
text.lower().split()
```
so yeah)